src/components/admin/Input/Input.tsx

```diff
@@ -62,6 +64,11 @@ export const Input: React.FC<InputProps> = ({
   ) : (
     <div className="Input">
       <div className={parentClasses}>
+        {label && (
+          <label className="block text-sm font-medium text-gray-700">
+            {label}
+          </label>
+        )}
```

Comment:
Please check the use of `<label>` tag through this PR. There are gotchas that need fixing:

- a `<label />` must be associated with an input
- a label will **at most** have one input associated, not two or three
- it can be associated if it wraps the input, like `<label ><input /> </label>`
- it can be associated by `id`, like  <label htmlFor={id}>


For the `id`, please use something like 
```tsx
  const id = useMemo(() => generateId("Input"), []);
```